.vsts/builds/e2e-tests.yml

```diff
@@ -0,0 +1,61 @@
+trigger: none
+phases:
+- phase: Host_Windows
+  queue:
+    name: DotNetCore-Windows
+    parallel: 4
+    matrix:
+      Portable:
+        Test.RuntimeIdentifier: none
+      SelfContainedWindows:
+        Test.RuntimeIdentifier: win-x64
+      SelfContainedLinux:
+        Test.RuntimeIdentifier: linux-x64
+      SelfContainedMacOs:
+        Test.RuntimeIdentifier: osx-x64
+  steps:
+  - task: NodeTool@0
+    displayName: Install Node 10.x
+    inputs:
+      versionSpec: 10.x
```

Comment:
Should this be 8.x? (per issues in templates that we won't fix)

Reply:
For now we can just test Node 10 on Windows.  I will create another PR to test both Node 8 and 10 on Windows.  Our 2.1 templates are only compatible with Node 8 on Mac.